examples/atmajs/index.html

```diff
@@ -11,34 +11,38 @@
 
 		TodoMVC Atma.js Application
 
-		1. Read readme.md - you will get basic information about the libraries
-		2. Hint: Viewing *.mask files enable javascript or less syntax highlighting in your IDE
+		1. Read the readme.md - you will get basic information about the libraries
+		2. Hint: Use Sublime/Atom plugins to highlight the *.mask syntax
+			, view them on github or enable Javascript/Less syntax for the extensions
```

Comment:
The positioning of this comma looks a little weird. Let's just keep comments as we'd write all-day sentences.